mllib/src/test/scala/org/apache/spark/ml/evaluation/ClusteringEvaluatorSuite.scala

```diff
@@ -66,16 +66,38 @@ class ClusteringEvaluatorSuite
     assert(evaluator.evaluate(irisDataset) ~== 0.6564679231 relTol 1e-5)
   }
 
-  test("number of clusters must be greater than one") {
-    val singleClusterDataset = irisDataset.where($"label" === 0.0)
+  /*
+    Use the following python code to load the data and evaluate it using scikit-learn package.
```

Comment:
Nit: I think you can use the standard block comment syntax here with leading asterisks

Reply:
this is the same as for `squaredEuclidean` where this format was used according to  https://github.com/apache/spark/pull/18538#discussion_r131100309

Reply:
I see, the idea is to make it more copy-pasteable. That's fine.